webshare/share-without-user-gesture.https.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WebShare Test: Share without user gesture error</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+  </head>
+  <body>
+    <script>
+        promise_test(t => {
+          return promise_rejects(
+              t, 'SecurityError',
```

Comment:
I filed a [bug about this](https://github.com/WICG/web-share/issues/45). Let's discuss there.